library/core/src/iter/adapters/peekable.rs

```diff
@@ -277,14 +276,13 @@ impl<I: Iterator> Peekable<I> {
     ///
     /// Consume any number less than 10.
     /// ```
-    /// #![feature(peekable_next_if)]
     /// let mut iter = (1..20).peekable();
     /// // Consume all numbers less than 10
     /// while iter.next_if(|&x| x < 10).is_some() {}
     /// // The next value returned will be 10
     /// assert_eq!(iter.next(), Some(10));
     /// ```
-    #[unstable(feature = "peekable_next_if", issue = "72480")]
+    #[stable(feature = "peekable_next_if", since = "1.49.0")]
```

Comment:
1.49 is currently in beta, this will be in 1.50.
```suggestion
    #[stable(feature = "peekable_next_if", since = "1.50.0")]
```